### PR TITLE
read me update

### PR DESCRIPTION
**Pull request by gitsierra**
## Purpose

Update readme with details on how to manually destroy resources from the terminal 

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Review by 13bfrancis — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,6 +60,14 @@ Running tests locally
 
 - npm test
 
+## Destroy
```
Comment (abelzer212) [inline README.md:63]: Maybe change to "Local Destroy Instructions"

File: README.md [hunk 2]
```diff
@@ -60,6 +60,14 @@ Running tests locally
...
+
+Want to destroy from your Mac? This may be necessary in some cases where you have orphaned infrastructure
```
Comment (abelzer212) [inline README.md:65]: Not everyone will be on a mac

File: README.md [hunk 3]
```diff
@@ -60,6 +60,14 @@ Running tests locally
...
+
+- Install/configure the AWS CLI (You will have to copy AWS keys into your terminal)
```
Comment (abelzer212) [inline README.md:67]: I'd expand this out to talk about the steps to retrieve those credentials from the cloudtamer site

File: README.md [hunk 4]
```diff
@@ -60,6 +60,14 @@ Running tests locally
...
+- brew install jq (if you don't have already installed)
```
Comment (abelzer212) [inline README.md:68]: What is jq? why are we installing it (some brief description will help give context)

File: README.md [hunk 5]
```diff
@@ -60,6 +60,14 @@ Running tests locally
...
+- sh destroy.sh "name_of_your_branch" and re-enter the branch name once it has found all associated resources
```
Comment (abelzer212) [inline README.md:69]: Are there any troubleshooting things to try if this command fails?

**Review by abelzer212 — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -60,6 +60,16 @@ Running tests locally
 
 - npm test
 
+## Local Destroy Instructions
```
Comment (jphechter) [inline README.md:63]: Can we change this slightly? To me "Local Destroy Instructions" sounds like it refers to something in my local environment. 

I don't have a perfect alternative, but maybe something like this?
- Destroy Orphaned AWS Infrastructure
- Destroy Infrastructure from Command Line

File: README.md [hunk 7]
```diff
@@ -60,6 +60,16 @@ Running tests locally
...
+
+Want to destroy locally? This may be necessary in some cases where you have orphaned infrastructure
+
+- Install/configure the AWS CLI (You will have to copy AWS keys into your terminal)
+- Go to https://cloudtamer.cms.gov/portal/project and login with your EUA
+- Go to the mdct-qmr-dev project and click on "Cloud access", click on your account, click on mdctqmr-developer-admin, click on Short-term Access Keys, Then paste the the text from Option 1 into your terminal
```
Comment (jphechter) [inline README.md:69]: I think these could be individual bullets

File: README.md [hunk 8]
```diff
@@ -60,6 +60,16 @@ Running tests locally
...
+- brew install jq (if you don't have already installed) (jq is command-line JSON processor necessary for the destroy script to run properly)
```
Comment (jphechter) [inline README.md:70]:
```suggestion
- `brew install jq` Install jq (command-line JSON processor). This is necessary for the destroy script to run properly.
```

File: README.md [hunk 9]
```diff
@@ -60,6 +60,16 @@ Running tests locally
...
+- sh destroy.sh "name_of_your_branch" and re-enter the branch name once it has found all associated resources. (There shouldn't be any errors but if there are any. Re-running the script should fix it)
```
Comment (jphechter) [inline README.md:71]:
```suggestion
- `sh destroy.sh name_of_your_branch` Run destroy script. You will be prompted to re-enter the branch name once it has found all associated resources. (There shouldn't be any errors but if there are any. Re-running the script should fix it)
```

**Review by jphechter — CHANGES_REQUESTED** [review]

**Review by jphechter — APPROVED** [review]
Love it! 👍

**Review by benjaminpaige — APPROVED** [review]

**Review by abelzer212 — APPROVED** [review]

**Pull request merged**